issue_template.md

```diff
@@ -1,3 +1,7 @@
+<!--
+Please only submit documentation-related issues with this form, or follow the contributing to OpenShift guidelines to submit a PR.
+-->
```

Comment:
Do we want a link with _contributing to OpenShift guidelines_ 
https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/contributing.adoc

Reply:
+1